frontend/src/components/layout/navigation/whatsnew/WhatsNewMenu.tsx

```diff
@@ -466,6 +489,53 @@ export const WhatsNewMenu = (props: Props) => {
     entries.push(phoneAnnouncement);
   }
 
+  const vpnAndRelayAnnouncement: WhatsNewEntry = {
+    title: l10n.getString("whatsnew-feature-bundle-header"),
+    snippet: l10n.getString("whatsnew-feature-bundle-snippet"),
+    content:
+      props.runtimeData && isBundleAvailableInCountry(props.runtimeData) ? (
+        <WhatsNewContent
+          description={l10n.getString("whatsnew-feature-bundle-body", {
+            monthly_price: getBundlePrice(props.runtimeData),
+            savings: "??%", // Design states 50%
+          })}
+          heading={l10n.getString("whatsnew-feature-bundle-header")}
+          image={BundleHero.src}
+          videos={{
+            // Unfortunately video files cannot currently be imported, so make
+            // sure these files are present in /public. See
+            // https://github.com/vercel/next.js/issues/35248
+            "video/webm; codecs='vp9'":
+              "/animations/whatsnew/bundle-promo-hero.webm",
+            "video/mp4": "/animations/whatsnew/bundle-promo-hero.mp4",
+          }}
+          cta={
+            <CtaLinkButton
+              // TODO: Add has_bundle to profile data => subscribed={props.profile.has_bundle}
```

Comment:
Waiting on a `profile.has_bundle` object to be built out, similar to `has_phone` so that we do not need to show the upgrade CTA to users who already have that plan.

Reply:
Could callout! Asked about that [on Slack](https://mozilla.slack.com/archives/C013CSYEL5T/p1663062840438759), since I'm not sure whether that's on anybody's radar.

Reply:
Thank you!

Reply:
OK, submitted https://github.com/mozilla/fx-private-relay/pull/2505, so we can add a check for `profile.has_premium && profile.has_vpn` in here (or possibly just the latter?).

Reply:
Awesome, I'll add a check for `profile.has_vpn`